lib/rules/jsx-sort-props.js

```diff
@@ -14,6 +14,19 @@ function isCallbackPropName(name) {
   return /^on[A-Z]/.test(name);
 }
 
+var RESEVED_PROPS_LIST = [
+  'children',
+  'dangerouslySetInnerHTML',
+  'htmlFor',
+  'key',
+  'ref',
+  'style'
+];
+
+function isReservedPropName(name, list) {
```

Comment:
this is just "is in list" if it takes the list argument; it'd probably be better to define this function so that `reservedList` is available via closure, and this function would take only the `name`?